src/components/LoginForm/index.js

```diff
@@ -68,14 +35,67 @@ function Login({ providers, ...props }) {
   }, [session]);
 
   return (
-    <Grid
-      container
-      justify="space-around"
-      alignItems="center"
-      className={classes.root}
-    >
-      <Grid item xs={12} className={classes.item}>
-        <form noValidate className={classes.formStyles}>
+    <Section className={classes.section}>
+      <Grid container>
+        <Grid item xs={7} className={classes.container}>
+          <Typography variant="h2">{title}</Typography>
+          <Typography className={classes.text}>{description}</Typography>
+          <form className={classes.form}>
+            <Grid container>
+              <Grid item xs={12}>
+                <TextField
```

Comment:
I don't think there is a need for a Grid here. You can use layout TextField directly, use form controls [directly](https://v4.mui.com/components/text-fields/#components) or use [`formik-mui`](https://github.com/stackworx/formik-mui) which should help with form processing as well (we used it in NigerianStat I believe).